.github/workflows/build-test.yml

```diff
@@ -32,56 +34,95 @@ jobs:
           path: "**/node_modules"
           key: ${{ steps.cache.outputs.primary-key }}
 
-  test-sdk:
-    name: "Test SDK"
+      - name: Save Cypress binary to cache
```

Comment:
remove this or keep? 

not relevant to this PR but at the same time we will need it for CI